src/site/content/en/media/video-and-source-tags/index.md

```diff
@@ -0,0 +1,361 @@
+  <img src="./accept-ranges-chrome-devtools.png" alt="Chrome DevTools screenshot: Accept-Ranges: bytes.">
+  <figcaption class="w-figcaption">Chrome DevTools screenshot: Accept-Ranges: bytes.</figcaption>
+</figure>
+
+### Include a poster image
+
+Add a poster attribute to the `video` element so that viewers have an idea of
+the content as soon as the element loads, without needing to download the video
+or start playback.
+
+```html
+<video poster="poster.jpg" ...>
+  ...
+</video>
+```
+
+A poster can also be a fallback if the video `src` is broken or if none of the
+supplied video formats are supported. The only downside to a poster images is an
+additional file request, which consumes some bandwidth and requires rendering.
+For more information see [Efficiently encode images](../uses-optimized-images/).
+
+
+<div class="w-columns">
+{% Compare 'worse' %}
+<figure class="w-figure" w-figure--inline-left>
+  <img src="./chrome-android-video-no-poster.png" alt="Without a fallback poster, the video just looks broken.">
+</figure>
+
+{% CompareCaption %}
+Without a fallback poster, the video just looks broken.
+{% endCompareCaption %}
+
+{% endCompare %}
+
+{% Compare 'better' %}
+<figure class="w-figure" w-figure--inline-right>
+  <img src="./chrome-android-video-poster.png" alt="A fallback poster makes it seem as if the first frame has been captured.">
+</figure>
+
+{% CompareCaption %}
+A fallback poster makes it seem as if the first frame has been captured.
+{% endCompareCaption %}
+
+{% endCompare %}
+</div>
+
+### Ensure videos don't overflow containers
+
+When video elements are too big for the viewport, they may overflow their
+container, making it impossible for the user to see the content or use the
+controls.
+
+<div class="w-columns">
+  <figure class="w-figure">
+    <img src="./chrome-android-portrait-video-unstyled.png" alt="Android Chrome screenshot, portrait: unstyled video element overflows
+    viewport.">
+    <figcaption class="w-figcaption">Android Chrome screenshot, portrait: unstyled video element overflows
+    viewport.</figcaption>
+  </figure>
+  <figure class="w-figure">
+    <img src="./chrome-android-landscape-video-unstyled.png" alt="Android Chrome screenshot, landscape: unstyled video element overflows
+    viewport.">
+    <figcaption class="w-figcaption">Android Chrome screenshot, landscape: unstyled video element overflows
+    viewport.</figcaption>
+  </figure>
+</div>
+
+You can control video dimensions using JavaScript or CSS. JavaScript libraries
+and plugins such as [FitVids](http://fitvidsjs.com/) (outside the scope fo this
+article) make it possible to maintain appropriate size and aspect ratio, even
+for videos from YouTube and other sources.
+
+For simple uses like the ones I'm describing here, use [CSS media
+queries](/web/fundamentals/design-and-ux/responsive/#css-media-queries) to
+specify the size of elements depending on the viewport dimensions; `max-width:
+100%` is your friend.
```

Comment:
```suggestion
You can control video dimensions using CSS. For simple uses like the ones
I'm describing here, use [CSS media
queries](/responsive-web-design-basics/#media-queries) to
specify the size of elements depending on the viewport dimensions; `max-width:
100%` is your friend.
```
Given that we know that JS bloat is such a big problem on the web, we should avoid recommending the use of JS for this.

Reply:
@rkochman What do you think?

Reply:
Can we recommend the CSS solution, but also note that javascript libraries are available if CSS doesn't meet all their needs?

Reply:
I'm going to do that and add a note about being careful of bloat.

Reply:
New language is here: https://github.com/GoogleChrome/web.dev/pull/3215/files#diff-441bd3256a136718e95146958fd6e11eR210